jerry-core/ecma/operations/ecma-objects.c

```diff
@@ -1164,6 +1164,11 @@ ecma_op_object_put_apply_receiver (ecma_value_t receiver, /**< receiver */
 
       /* 5.e.iv */
       result = ecma_op_object_define_own_property (receiver_obj_p, property_name_p, &prop_desc);
+
+      if (JERRY_UNLIKELY (ecma_is_value_false (result)) && is_throw)
+      {
+        result = ecma_raise_type_error (ECMA_ERR_MSG ("Proxy trap returned falsish"));
```

Comment:
Proxy put trap returned with false.

Reply:
That's a matter of taste. Your suggestion is equivant to the SpiderMonkey's error message mine is to V8. To keep the consistecy, I'd leave the V8 error message.

Reply:
Consistency?